bin/natsboard

```diff
@@ -1,3 +1,7 @@
 #!/usr/bin/env node
 
-require('..');
\ No newline at end of file
+require('..');
+
+process.on('SIGINT', function() {
+    process.exit();
+});
```

Comment:
I needed to add this, otherwise `ctrl+c` wasn't working with the running container.